engine/lib/attr_payloads.ml

```diff
@@ -6,7 +6,18 @@ let payloads : attrs -> (Types.ha_payload * span) list =
   let parse =
     (* we have to parse ["JSON"]: first a string, then a ha_payload *)
     function
-    | `String s -> Yojson.Safe.from_string s |> Types.parse_ha_payload
+    | `String s -> (
+        match Yojson.Safe.from_string s |> Types.safe_ha_payload_of_yojson with
+        | Error _ ->
+            Stdlib.prerr_endline
+              [%string
+                {|
+The hax engine could not parse a hax attribute.
```

Comment:
Couldn't this also happen if the hax_lib is right but somehow the content of the attibute is malformed?

Reply:
No, this situation should never happen, the hax lib attributes are always JSON serializations of a Rust enum, and the OCaml parser used here is directly generated from the Rust enum definition.

So, we could still have a bug in generated parsers, but this almost never happened (it happened once because of OCaml ints being smaller than Rust ints)